scripts/sdk_process/patch_models.py

```diff
@@ -119,7 +114,6 @@ def solve_one_model(models_module, output_folder, track2=False):
         enum_file_module_name = None
 
     write_model_file(Path(output_folder, "models_py3.py"), models_classes, track2=track2)
-    write_model_file(Path(output_folder, "models.py"), py2_models_classes, track2=track2)
```

Comment:
I believe most of CLI code still uses `models.py`, we cannot delete this line until we change all code to use `models_py3.py`.

Reply:
I think CLI only uses `models_py3.py`, not `models.py`. CLI simply imports from `models/__init__.py` and `py2` or `py3` is automatically decided.

https://github.com/Azure/azure-sdk-for-python/blob/ad5308d9befefdc0cdc61c4a47a3329fd874b8a0/sdk/network/azure-mgmt-network/azure/mgmt/network/v2020_08_01/models/__init__.py#L9-L12

```py
try:
    from ._models_py3 import Alias
    from ._models_py3 import AliasPath
```

Reply:
It should be ok since the test pass.

Reply:
Sorry, I misrecognized this file as `build_scripts/windows/scripts/patch_models_v2.py` and I was referring to this piece of logic during MSI build to remove all `*_py3` model files:
https://github.com/Azure/azure-cli/blob/ada48783a896158e124a751d79dc2839fcf87cd0/build_scripts/windows/scripts/build.cmd#L113-L123

Reply:
Shouldn't we only use `py3` files in MSI now?